src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -100,6 +100,11 @@ public abstract class Submission extends DomainObject implements Comparable<Subm
     @Column(name = "submission_date")
     private ZonedDateTime submissionDate;
 
+    @OneToMany(mappedBy = "submission")
```

Comment:
We often use `cascade = CascadeType.REMOVE, orphanRemoval = true`. I think this should also be used here